cmd/svcat/output/binding.go

```diff
@@ -50,18 +61,54 @@ func WriteBindingList(w io.Writer, bindings ...v1beta1.ServiceBinding) {
 		"Status",
 	})
 
-	for _, binding := range bindings {
+	for _, binding := range bindingList.Items {
 		t.Append([]string{
 			binding.Name,
 			binding.Namespace,
 			binding.Spec.ServiceInstanceRef.Name,
 			getBindingStatusShort(binding.Status),
 		})
 	}
-
 	t.Render()
 }
 
+// WriteBindingList prints a list of bindings in the specified output format.
+func WriteBindingList(w io.Writer, outputFormat string, bindingList *v1beta1.ServiceBindingList) {
+	switch outputFormat {
+	case "json":
```

Comment:
Would you please switch these to use a const so that I don't accidentally fat finger it later? 😀